JavaScript/JavaScript.sublime-syntax

```diff
@@ -207,13 +207,13 @@ contexts:
         8: storage.type.function.arrow.js
     - match: |-
         (?x)
-        (\b_?[A-Z][$\w]*)?
+        (\b_?[a-zA-Z_?.$]*)?
```

Comment:
I was just going to comment I have no idea about the question mark. I was just trying to search to find if it is some upcoming JavaScript feature, but I couldn't find anything. I am guessing that was just copied over from the javascript next package when it was originally copied into Sublime?